### PR TITLE
Add recipe for Tiroler Gröstl.

### PR DESCRIPTION
**Pull request by printplaatreparatie**
I'm not from Austria, maybe there are locals from Tirol that can improve
on this already delicious recipe?

### CODE REVIEW

File: src/tiroler-grostl.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- ⏲️  Prep time: 15 min
+- 🍳 Cook time: 30 min
+- 🍽️ Servings: 5
+
+## Ingredients
+
+- 1 kg / 2 lbs potatoes
+- 2 red onions
+- 500 gr / 1 lbs porkchops
+- 400 gr / 14 ounce bacon (smoked)
```
Comment (hallzy) [inline src/tiroler-grostl.md:12]:
```suggestion
- 500 g / 1 lbs porkchops
- 400 g / 14 ounce bacon (smoked)
```

File: src/tiroler-grostl.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+## Directions
+
+1. Clean potatoes and boil with skin.
+2. Chop onion, squeeze and mince the garlic.
+3. Dice pork and bacon.
+4. Slice cooked potatoes in thick wedges.
+5. Fry potato wedges in butter, add more butter while stirring.
+6. Add pork and bacon and bake until crispy.
+7. Add and stir the rest of the ingredients, except eggs.
+8. Fry the eggs seperately and serve on top.
```
Comment (hallzy) [inline src/tiroler-grostl.md:27]:
```suggestion
8. Fry the eggs separately and serve on top.
```

**Review by hallzy — CHANGES_REQUESTED** [review]
Any chance you have an image you can attach?

File: src/tiroler-grostl.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Contribution
+
+ - Peter Wilbrink -- [printplaatreparatie.nl](https://printplaatreparatie.nl)
```
Comment (Socerest2) [inline src/tiroler-grostl.md:31]:
```suggestion
 - Peter Wilbrink -- [website](https://printplaatreparatie.nl)
```

**Review by Socerest2 — CHANGES_REQUESTED** [review]

File: src/tiroler-grostl.md (hunk 3 above)
Comment (printplaatreparatie) [reply src/tiroler-grostl.md:31]: Is this mandatory? It's not a personal website.

**Comment by printplaatreparatie** [comment]
@hallzy Unfortunately I don't have a picture. Feel free to make some Gröstl and add an image of your succesful meal.😊

**Review by hallzy — APPROVED** [review]

**Pull request merged**